src/applications/mhv-medical-records/components/CareSummaries/AdmissionAndDischargeDetails.jsx

```diff
@@ -142,31 +142,40 @@ ${record.summary}`;
         <h3 className="vads-u-font-size--base vads-u-font-family--sans">
           Location
         </h3>
-        <p data-testid="note-record-location"> {record.location}</p>
+        <p data-testid="note-record-location" data-dd-privacy="mask">
+          {' '}
+          {record.location}
+        </p>
```

Comment:
I'm not sure why this had a space in there, but let's get rid of it. It shouldn't be there.

Reply:
fixed